privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -345,7 +351,20 @@ def do(self, action, options=None):
                 mimetype = handler_options.get("mimetype", "plain")
                 useremail = recipient.get("email")
                 reply_to = handler_options.get("reply_to")
-
+                attachment = handler_options.get("attachment", False)
```

Comment:
If "attachment" would not exist, then ``attachment`` will be None and then below ``if attachment`` will also work.
Is there a reason for defaulting it to False instead of None?

Reply:
Just a precaution and making things more clear. If You like i can remove it.

Reply:
For me it would be clearer the other way round.
But I am not sure, if we need certain styles at this point.